src/ducks/initialize.js

```diff
@@ -0,0 +1,60 @@
+import { fetchProject } from './project';
+import { fetchWorkflow } from './workflow';
+import { fetchGuide } from './field-guide';
+
+const FETCH_RESOURCES = 'FETCH_RESOURCES';
+const FETCH_RESOURCES_SUCCESS = 'FETCH_RESOURCES_SUCCESS';
+const FETCH_RESOURCES_ERROR = 'FETCH_RESOURCES_ERROR';
+
+const INITIALIZER_STATUS = {
+  IDLE: 'initializer_status_idle',
+  FETCHING: 'initializer_status_fetching',
+  READY: 'initializer_status_ready',
+  ERROR: 'initializer_status_error'
+};
+
+// Reducer
+const initialState = {
+  ready: false,
+  status: INITIALIZER_STATUS.IDLE
+};
+
+const initializeReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_RESOURCES:
+      return Object.assign({}, state, {
+        status: INITIALIZER_STATUS.FETCHING
+      });
+
+    case FETCH_RESOURCES_SUCCESS:
+      return {
+        status: INITIALIZER_STATUS.READY,
+        ready: true
+      };
+
+    default:
+      return state;
+  }
+};
+
+const fetchResources = () => {
+  return (dispatch) => {
+    dispatch({ type: FETCH_RESOURCES });
+
+    Promise.all([
```

Comment:
Following my comments in PR #16, we need to keep an eye on `fetchWorkflow()`. If I understand correctly, Promise.all() will... run all the promises in parallel instead of in sequence, if I'm correct? (Someone check me here; I'll try to write a quick test to be sure.)

In any case, I don't see a guarantee that we'll get the Workflow ID from the Project resource _before_ we get the Project promise resolves; so to avoid using a hardcoded Workflow ID, we might still need to chain it like `fetchProject().then((proj)=>{ fetchWorkflow(proj.workflowid) })`